controller/iteration_blackbox_test.go

```diff
@@ -399,7 +445,11 @@ func (rest *TestIterationREST) TestFailShowIterationMissing() {
 
 func (rest *TestIterationREST) TestSuccessUpdateIteration() {
```

Comment:
he we could have `TestUpdateIteration()` with some nested tests (`ok` / `fail`)

Reply:
Yes I have done nesting of sub-tests , please see https://github.com/fabric8-services/fabric8-wit/pull/1783/commits/80ae86445281239f604d0a038eba40ff8b347607

Reply:
ok